tools/serve_header/serve_header.py

```diff
@@ -240,7 +240,7 @@ def stop(self):
         self.observer.stop()
         self.observer.join()
 
-class HeaderRequestHandler(SimpleHTTPRequestHandler):
+class HeaderRequestHandler(SimpleHTTPRequestHandler): # lgtm[py/missing-call-to-init]
```

Comment:
Why don't you call `SimpleHTTPRequestHandler`'s `__init__` function?

Reply:
I do. `BaseRequestHandler` doesn't and that seems to be a deliberate choice.

> Class HeaderRequestHandler may not be initialized properly as method BaseRequestHandler.__init__ is not called from its __init__ method.